### ISSUE TITLE
Double check interpretation of Chi Squared

### ISSUE THREAD

**Issue by CATower** [pappaburger/intro-data-capstone-biodiversity#5]
For the chi-squared test comparing reptile and mammal percentages, you state in the presentation that more care should be spent on reptiles. Remember that a chi-squared test tells you that there is a significant difference between the groups, but its not directional so its up to you to determine which has the higher percentage. Mammals have the higher percentage of being endangered so that is where conservationists should focus.

**Comment by pappaburger** [comment]
The percentage of mammal protected is 17% while that for reptile is 6.94%. Should reptiles need more protection?
Thank you

**Comment by CATower** [comment]
That's correct about the numbers. Keep in mind that the percentage is all the different levels of endangered and receive protected status. This shows us that it is more likely for a mammal to need to be protected in the future.
